indico/modules/events/papers/client/js/components/Paper.jsx

```diff
@@ -59,3 +85,54 @@ Paper.propTypes = {
   eventId: PropTypes.number.isRequired,
   contributionId: PropTypes.number.isRequired,
 };
+
+function PaperSteps({isAccepted, hasEditable, fileTypes, existingFiles}) {
+  const {event, contribution} = useSelector(getPaperDetails);
+
+  return (
+    <Step.Group ordered fluid>
+      <Step completed={isAccepted}>
+        <Step.Content>
+          <Step.Title>Peer Reviewing</Step.Title>
+          <Step.Description style={{marginBottom: '0'}}>
+            {isAccepted ? (
+              <Translate>The paper was accepted</Translate>
```

Comment:
I used `was` because `has been` looks more like something that happened just now, an immediate warning. While the first refers to a past action, can be an hour, day, week.